demos/continuous_batching/scaling/README.md

```diff
@@ -52,35 +67,28 @@ docker run -v $(pwd)/nginx.conf:/etc/nginx/nginx.conf:ro -d --net=host -p 80:80
 
 Start benchmarking script like in [demo](../README.md), pointing to the load balancer port and host.
 ```bash
-python benchmark_serving.py --host localhost --port 80 --endpoint /v3/chat/completions --backend openai-chat --model meta-llama/Meta-Llama-3-8B-Instruct --dataset-path ShareGPT_V3_unfiltered_cleaned_split.json --num-prompts 2000 --request-rate inf --save-result --seed 10
+python benchmark_serving.py --host localhost --port 80 --endpoint /v3/chat/completions --backend openai-chat --model meta-llama/Meta-Llama-3-8B-Instruct --dataset-path ShareGPT_V3_unfiltered_cleaned_split.json --num-prompts 6000 --request-rate 20
```

Comment:
Why do we change request rate. TTFT is still high then why not go with inf for max throughput?

Reply:
@mzegla  with 6000 requests sent simultaneously I observed issues with nginx and some requests were dropped on the way